### ISSUE TITLE
Improvements: type get(decrypt) ciphertext, get vcard and get multi_vcard

### ISSUE THREAD

**Issue by phantom-limb** [open-wa/wa-automate-nodejs#56]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.8.47](https://github.com/open-wa/wa-automate-nodejs/compare/1.8.46...1.8.47) (2020-04-10)

- implemented `getVCards`. resolves #56 [`#56`](https://github.com/open-wa/wa-automate-nodejs/issues/56)
- Release 1.8.47 [`a6a665e`](https://github.com/open-wa/wa-automate-nodejs/commit/a6a665ed31d5369ab93cd9ee8f670cdc17efa40e)

**Comment by smashah** [comment]
@phantom-limb you can now extract vcards:

https://open-wa.github.io/wa-automate-nodejs/classes/whatsapp.html#getvcards

please make a new issue for ciphertext if still required. Thanks
